screenpy/actions/see_all_of.py

```diff
@@ -54,10 +56,12 @@ def perform_as(self: SelfSeeAllOf, the_actor: Actor) -> None:
     def __init__(self: SelfSeeAllOf, *tests: T_T) -> None:
         for tup in tests:
             if isinstance(tup, tuple):
-                if len(tup) != 2:
-                    raise UnableToAct("Tuple must contain Question and Resolution.")
+                if len(tup) != 2:  # noqa: PLR2004
```

Comment:
We have a lot of places, 5 i think, where we handle allowing users to pass in 3 different types of data—a dictionary, a list of tuples, or a paired list. We might want to make some sort of "validate pairwise argument" or something.

Anyway, `PLR2004` says that the 2 is a magic number instead of a constant... i mean of course it is.

Reply:
Yeah, I played with that rule and thought it should be disabled. I get what it was going for, but not everything is a magic number situation.

Reply:
I think i'd prefer to have it remind us just in case, that's why i'm leaving it in for now. That way we can mark the spots to make it clear we mean for this value to be there.